README.md

```diff
@@ -22,13 +22,17 @@ Full documentation [available here](https://diode.suzaku.io).
 
 Add following dependency declaration to your Scala project.
 
-<pre><code class="lang-scala">"io.suzaku" %% "diode" % "1.1.1"</code></pre>
+```scala
+"io.suzaku" %% "diode" % "1.1.1"
+```
 
 In a Scala.js project the dependency looks like this.
 
-<pre><code class="lang-scala">"io.suzaku" %%% "diode" % "1.1.1"</code></pre>
+```scala
+"io.suzaku" %%% "diode" % "1.1.1"
+```
 
-<img align="right" src="doc/images/architecture.png">
+![Diode architecture](doc/images/architecture.png)
 
```

Comment:
This would lose the right alignment style which places the image to the right side of the text.